src/components/switch/switch-theme.scss

```diff
@@ -9,6 +9,14 @@ md-switch.md-THEME_NAME-theme {
     background-color: '{{background-500}}';
   }
 
+  &.md-focused:not([disabled]) {
```

Comment:
Based on https://material.io/archive/guidelines/components/selection-controls.html#selection-controls-switch, it seems like the thumb needs to be displayed when focused and disabled. This can't be done with tabbing, but it is possible to focus a disabled switch when using a screen reader or other assistive device.

We should probably remove this disabled check.

Reply:
I did some more testing and the following seemed to work best for me:
```scss
  &.md-focused {
    &:not(.md-checked) {
      .md-thumb:before {
        background-color: '{{foreground-4}}';
      }
    }
    &[disabled] {
      .md-thumb:before {
        background-color: '{{foreground-4}}';
      }
    }
  }
```